defaults.lua

```diff
@@ -115,10 +115,10 @@ DCREREADER_CONFIG_MARGIN_SIZES_SMALL = {5, 10, 5, 10}
 DCREREADER_CONFIG_MARGIN_SIZES_MEDIUM = {10, 15, 10, 15}
 DCREREADER_CONFIG_MARGIN_SIZES_LARGE = {20, 20, 20, 20}
 
--- crereader font gamma
-DCREREADER_CONFIG_LIGHTER_FONT_GAMMA = 10
-DCREREADER_CONFIG_DEFAULT_FONT_GAMMA = 15
-DCREREADER_CONFIG_DARKER_FONT_GAMMA = 25
+-- crereader font gamma (no more used)
```

Comment:
So why keep it? :-P

Reply:
Just so people don't someday go there wondering "I'm sure there was a setting for that!, where is it?!". They'll find them commented, and know they are no more used. That's friendly.
You want them removed?

Reply:
Nah, keep it but change it to "no longer used" or "not used anymore"